src/coreclr/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -805,5 +805,10 @@ public static object BindToMoniker(string monikerName)
 
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern IntPtr GetFunctionPointerForDelegateInternal(Delegate d);
+
+#if DEBUG // Used for testing in Checked or Debug
+        [DllImport(RuntimeHelpers.QCall)]
+        internal static unsafe extern delegate* unmanaged[Stdcall]<int> GetIsInCooperativeGCModeFunctionPointer();
```

Comment:
I don't think there is any reason for the `[Stdcall]` as it is a platform API and will always be the default.